robot_param: {
   EncoderCprLeft: 400,
   EncoderCprRight: 400,
   EncoderPosition: 0,
   Height_mm: 200,
   Lenght_mm: 365, 
   MaxAmpereMotorLeft_mA: 1650, 
   MaxAmpereMotorRight_mA: 1650, 
   MinChargedBatteryLevel_mV: 12000,
   MaxChargedBatteryLevel_mV: 16800,
   MaxRpmMotorLeft: 65535, 
   MaxRpmMotorRight: 65535, 
   MaxTorqueMotorLeft_Ncm: 60, 
   MaxTorqueMotorRight_Ncm: 60,
   MotorEnableLevel: 1, 
   RatioMotorLeft: 55, 
   RatioMotorRight: 55,
   RatioShaftLeft: 3,
   RatioShaftRight: 3,
   Weight_g: 5000, 
   WheelBase_mm: 340, 
   WheelRadiusLeft_cent_mm: 3500,
   WheelRadiusRight_cent_mm: 3500, 
   Width_mm: 420
}
